docs-2.0/3.ngql-guide/10.tag-statements/2.drop-tag.md

```diff
@@ -0,0 +1,29 @@
+# DROP TAG
+
+```ngql
+DROP TAG [IF EXISTS] <tag_name>
+```
+
+`DROP TAG` drops a tag with the given name in a graph space. You must have the `DROP` privilege for the graph space. (TODO: privilege doc) To drop a tag in a specific graph space, you must use the graph space first.
+
+Before you drop a tag, make sure that the tag does not contain any indexes or TTLs. If a tag contains any indexes or TTLs, a conflict error occurs when you drop it.
+
+A vertex can have one or more tags. If a vertex has only one tag, after you drop it, the vertex **CANNOT** be accessible. But its edges are available. The vertex is deleted in next compaction. If a vertex has multiple tags, after you drop one of them, the vertex is still accessible. But all the properties defined by this dropped tag are not accessible.
+
+This operation only deletes the Schema data, all the files and directories in the disk are NOT deleted directly, data is deleted in the next compaction.
```

Comment:
This operation only deletes the schema data, and all the files and directories in the disk are NOT deleted directly. Data will be deleted in the next compaction.